### DISCUSSION TITLE
Dumb question about privacy

### DISCUSSION THREAD

**Discussion by ghost** [moneymanagerex/moneymanagerex#4756] [Q&A] [answered]
Hi, 

I'm new to this program, and sorry for the somewhat silly question: Does  Money Manager EX automatically sync/upload data to a centralized server somewhere without my consent? Or is data sync completely optional (and hence, the app can run in a completely air-gapped environment)?

Thanks in advance.

**Comment by tactilis** [comment, marked as answer]
> Does Money Manager EX automatically sync/upload data to a centralized server somewhere without my consent?

No.

MMEX just writes to its .mmb database file. If you want to host this in the cloud or sync it with another device, then you must arrange this outside of MMEX.

> the app can run in a completely air-gapped environment

Yes

There are 3 options that you may wish to turn off:
 
![image](https://user-images.githubusercontent.com/4060092/176464507-dcf1c6f0-55d6-4f42-8517-844107b4708e.png)
 
But none of these relate to your financial data.

**Reply by ghost** [reply]
Thanks for clarifying!
